doc/customize/icons.md

```diff
@@ -24,6 +24,8 @@ Pictograms can be linear or filled.
 - Linear pictograms have to be used at size 20px or bigger for Accessibility reasons. Typical use cases include Primary and Secondary buttons.
 - Filled pictograms have to be used at size 20 px or smaller. Typical use cases include Tertiary and Quaternary buttons.
 
+Always download icons from the 'Fondation' figma so the ViewBox/Path ratio is consistent. 
+
```

Comment:
Possible d'y mettre un lien ?

https://www.figma.com/file/jswn3bl2Sy7hm4XDyOLWE9/Fondations

Reply:
Je pense qu'on peut clarifier aussi le rapport ViwBox/Path que tu évoques.


```suggestion
If you don't download icons from the Fondations file, make sure to use the viewbox values from your import, rather than the default "48 48 0 0" used in Fondations. The Path and the ViewBox are related to maintain a consistent ratio. 
```

Tu penses que c'est assez clair ?